tests/cloud_functions/testing_utils.py

```
@@ -0,0 +1,226 @@
+        A tuple containing (db, user_ids) where db is the InMemoryFirestoreClient
+        and user_ids is a list of generated user IDs.
+    """
+    if db is None:
+        db = InMemoryFirestoreClient()
+
+    user_ids = []
+    for idx in range(num_users):
+        user_id = str(uuid.uuid4()).replace("-", "")[:28]
+        user_ids.append(user_id)
+
+        now = datetime.now().replace(tzinfo=datetime.timezone.utc)
```

Ruff (0.8.2)

`datetime.datetime.now()` called without a `tz` argument

(DTZ005)